staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -50,6 +50,11 @@ type PriorityClass struct {
 	// when this priority class should be used.
 	// +optional
 	Description string `json:"description,omitempty" protobuf:"bytes,4,opt,name=description"`
+
+	// nonPreempting specifies whether a pod with this PriorityClass could trigger a preemption process.
+	// If this field is unset, the PriorityClass is considered a preempting class by default.
+	// +optional
+	NonPreempting bool `json:"nonPreempting,omitempty" protobuf:"bytes,5,opt,name=nonPreempting"`
```

Comment:
v1alpha1 PrioirtyClass is deprecated by newer versions. We don't need to add the field to this version.